x-pack/legacy/plugins/monitoring/public/components/no_data/no_data.js

```diff
@@ -54,6 +55,14 @@ export function NoData(props) {
   if (useInternalCollection) {
     return (
       <EuiPage>
+        <EuiScreenReaderOnly>
+          <h1>
+            <FormattedMessage
+              id="xpack.monitoring.no_data.internal_collection.heading"
+              defaultMessage="No monitoring data found. (Internal collection)"
```

Comment:
Do we need to say `(Internal collection)` here?

Reply:
Removed